content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -19,17 +19,21 @@ aliases:
 test: n/a
 ---
 
-When configuring a production deployment of Istio, you need to answer a number of questions.
-Will the mesh be confined to a single {{< gloss >}}cluster{{< /gloss >}} or distributed across
-multiple clusters? Will all the services be located in a single fully connected network, or will
-gateways be required to connect services across multiple networks? Is there a single
-{{< gloss >}}control plane{{< /gloss >}}, potentially shared across clusters,
-or are there multiple control planes deployed to ensure high availability (HA)?
-If there is more than one cluster being deployed, and more specifically in isolated networks,
-are they going to be connected into a single {{< gloss >}}multicluster{{< /gloss >}}
-service mesh or will they be federated into a {{< gloss >}}multi-mesh{{< /gloss >}} deployment?
-
-All of these questions, among others, represent independent dimensions of configuration for an Istio deployment.
+When configuring a production deployment of Istio, you need to answer a number
+of questions. Will the mesh be confined to a single
+{{< gloss >}}cluster{{< /gloss >}} or distributed across multiple clusters? Will
+all the services be located in a single fully connected network, or will
+gateways be required to connect services across multiple networks? Is there a
+single {{< gloss >}}control plane{{< /gloss >}}, potentially shared across
+{{< gloss "remote cluster" >}}remote clusters{{< /gloss >}}, or are there
+multiple {{< gloss "primary cluster" >}}primary clusters{{< /gloss >}} deployed
```

Comment:
Does "primary cluster" just mean a cluster with its own control plane or something else? IIUC this is trying to ask "is there a single control plane shared across multiple clusters, or does each cluster have its own control plane for HA?"

Reply:
@ayj yes, a primary cluster is exactly that and a remote cluster is a cluster connected to a control plane outside of the cluster. We defined those 2 terms in a previous PR and they are now in the glossary (https://preliminary.istio.io/latest/docs/reference/glossary/#P).

That said, I don't think this overview paragraph should be changed. It is talking about the high level organization of clusters and control planes. The primary/remote distinction is deployment specific, so not appropriate here. A 2 cluster mesh with a single control plane could be either 1 primary cluster and 1 remote cluster, or 2 remote clusters with an external managed control plane. Same for multiple control planes, they could be multiple primary clusters, or a mix of primary and remote cluster, managed control planes, etc. Bottom line is that the primary/remote cluster terminology does not belong in this paragraph.

Reply:
@frankbu It does belong in the intro since readers typically have knowledge of clusters but not control planes. We need to pick up readers where they are most likely at (Istio readers typically have some familiarity with Kubernetes and clusters). This paragraph is not meant to provide readers with all the possibilities, instead it aims to entice them to continue reading.

Reply:
I'm fine with Primary/Remote cluster terminology. I thought phrases such as "are there multiple primary clusters deployed" and "is there one cluster being deployed" may be confusing to readers. To me it implies we're talking about deploying Kubernetes clusters themselves, and not necessarily configuring/deploying Istio in those clusters. 

How about the following? "Is there a single control plane, potentially shared across remote clusters, or are there multiple primary clusters for HA, each with their own control plane?"

Reply:
I'm also fine with weaving in the primary/remote cluster terminology, but we shouldn't change the high level overview question here: do you have one or multiple control planes? Note that multiple primary clusters != multiple control planes. Multiple primary clusters is one specific implementation approach.